boa/core/test.py

```diff
@@ -352,6 +352,26 @@ def construct_metadata_for_test(recipedir_or_package, config):
     return m, hash_input
 
 
+def test_run_v2(test_yaml_path,):
+    PREFIX = "/Users/madhur/mambaforge/envs/boa/"
+    import ruamel.yaml as yaml
+    from rich.console import Console
+
+    console = Console()
+
+    f = open(test_yaml_path)
+    test_yaml = yaml.safe_load(f)
+
+    exists = test_yaml["exists"] if "exists" in test_yaml else None
```

Comment:
you can use `test_yaml.get("exists")` that will return either the value or `None`.

Reply:
Done